### DISCUSSION TITLE
Possible New WIFI Exploit

### DISCUSSION THREAD

**Discussion by gambitin** [3kh0/ext-remover#1146] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I was recently using the 124+ incognito exploit and the captive portal didn't pop up, this was strange because I had the flag enabled. Instead, when I switched to detectportal.firefox.com, and used that network when logged in, everything on that network was unblocked, unless blocked like this. All extensions were dead on that network and I could access the teacher hapara dashboard.
![image](https://github.com/user-attachments/assets/e302a5d6-02ee-444a-9278-1a763828a661)

**Comment by Balistic123** [comment]
can you provide a more detailed instructions with the incognito exploit too?

**Comment by S-PScripts** [comment]
? Did you use my guide at https://s-pscripts.github.io/incognito-v123/

**Reply by gambitin** [reply]
No, I was trying to, but I did not have it work, instead, I got a non-incognito window with all extensions in a dead state.
